src/librawspeed/common/DngOpcodes.h

```diff
@@ -113,10 +113,11 @@ class OpcodeFixBadPixelsConstant: public DngOpcode
 {
 public:
   OpcodeFixBadPixelsConstant(const uchar8* parameters, uint32 param_max_bytes, uint32 *bytes_used);
-  virtual ~OpcodeFixBadPixelsConstant() = default;
+  ~OpcodeFixBadPixelsConstant() override = default;
```

Comment:
A sub-class destructor always overrides it's base class destructor. Providing that keyword on a destructor has no meaning (I did not even know this compiles).